pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -280,6 +290,10 @@ func (c *AuthflowController) HandleResumeOfFlow(
 }
 
 func (c *AuthflowController) HandleStep(w http.ResponseWriter, r *http.Request, handlers *AuthflowControllerHandlers) {
+	if handled := c.handleInlinePreviewIfNecessary(w, r, handlers); handled {
+		return
+	}
```

Comment:
Not sure if it is the best way to handle this.

Reply:
Looks good to me.